se_feedback/sprint_6/feedback_sprint_6_all.md

```diff
@@ -61,3 +61,23 @@ Total: ****
 | Code Review            |     |
 
 ##Notes
+* ADD
+	* The diagrams makes your explanation a lot clearer.
+	* You should explain how you run the tools in the Concurrency section 
+* Good job on the test coverage document!
+* Java
+	* `JavaController.setASATOutput`
+		* Consider using a properties file (or JSON) to store ASAT Output locations
```

Comment:
I currently don't see how this might be useful? Because we will first have to write the locations of ASAT output and then reading it back again? 

Or have I misunderstood what you meant?